src/ZPublisher/tests/test_http_header_utils.py

```diff
@@ -0,0 +1,37 @@
+##############################################################################
+#
+# Copyright (c) 2017 Zope Foundation and Contributors.
```

Comment:
This would be a good thing to add to the newly created `contributing` file. What do you have to do when you create a new file/ module?

Reply:
```suggestion
# Copyright (c) 2020 Zope Foundation and Contributors.
```